content/workload-reduction-toolkit/address-workload-issues/feedback-and-marking/review-and-streamline-feedback-and-marking.html.erb

```diff
@@ -0,0 +1,136 @@
+                      </p>
+                      <p class="govuk-body-s govuk-!-margin-bottom-6">
+                        <strong>Resource type:</strong> Presentation
+                      </p>
+                      <p class="govuk-body-s govuk-!-margin-bottom-6">
+                        <strong>File type:</strong> OpenDocument Presentation, 171KB, 7 pages
+                      </p>
+                      <a class="govuk-link govuk-link--no-visited-state govuk-!-font-weight-bold"
+                        href="<%= @base_url %>/assets/files/Review and streamline feedback and marking.odp">
+                        Download
+                      </a>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </ul>
+      </div>
+
+      <div class="dfe-width-container govuk-grid-row">
+        <h2 class="govuk-heading-m govuk-!-margin-top-7 govuk-!-margin-bottom-7">
+          Audience
+        </h2>
+
+        <p>
+          This resource has been provided to support senior leadership teams to run a session with the whole school, teams or departments.   
+        </p>
+      </div>
+
+      <div class="dfe-width-container govuk-grid-row">
+        <h2 class="govuk-heading-m govuk-!-margin-top-7 govuk-!-margin-bottom-7">
+          Presentation length
+        </h2>
+
+        <p>
+          The presentation should take 1 hour and 15 minutes. You should also allocate time for the facilitator to prepare the
+          session for use in your school.
+        </p>
+      </div>
+
+      <div class="dfe-width-container govuk-grid-row">
+        <h2 class="govuk-heading-m govuk-!-margin-top-7 govuk-!-margin-bottom-7">
+          How to run with your staff
+        </h2>
+
+        <p>
+          Depending on the situation in your school, you might wish to use this workshop as a starting point to initiate further exploration, or to prepare for the workshop with activities beforehand.  
+        </p>
+        <p>
+          This could include your senior leadership team discussing: 
+            <ul>
+                <li>to what extent teachers follow your existing policy – and how you know?</li>
+                <li>can you clearly articulate what you expect from teachers – and is it reasonable?</li>
+                <li>do you have evidence to support the practices in your current policy?</li>
+                <li>how does your policy compare with examples from other schools?</li>
+            </ul>
+        </p>
+        <p>
+          Senior leadership teams are likely to want to oversee the process although they could also consider if it is more appropriate for another member of staff to facilitate the session and lead on follow-up. For example, a head of department. Alternatively, you might wish to consider asking somebody independent to facilitate the workshop to free up time to take part.  
```

Comment:
The first sentence here is a bit too long:
'Senior leadership teams are likely to want to oversee the process.  However they may want to consider if another staff member could facilitate the session and lead on follow-up, for example, a head of department. Alternatively, you might want to consider asking an independent person to facilitate the workshop, to free up time to take part.'